### ISSUE TITLE
OSGi version declaration using "o" (letter o) not "0" (zero)

### ISSUE THREAD

**Issue by joakime** [eclipse/jetty.project#2656]
Possible bug in OSGi version declaration?

https://github.com/eclipse/jetty.project/blob/71dc6d9cebfb2a4c78df7affd2a9b8af0bc5969c/jetty-osgi/test-jetty-osgi-context/pom.xml#L74-L78

On line 76

The `1.0.o` (using "o" not zero)?

**Comment by joakime** [comment]
Some of the other osgi test poms have similar version declaration quirks.

**Comment by janbartel** [comment]
Yeah, probably just delete those Import-Package statements altogether and let the bnd plugin generate them. I've created PR #2657
